lib/ecto/log_entry.ex

```diff
@@ -89,4 +103,45 @@ defmodule Ecto.LogEntry do
       []
     end
   end
+
+  defp colorize({entry, data}) do
+    color = entry
+    |> sql_color
+    |> to_ansicolor
+
+    # We need to prepend the list with the color
+    # and append a clear color to reset.
+    data = data
```

Comment:
Instead of parsing the query with a regular expression, I believe we should get the query from the entry struct and just match on its prefix:

    defp sql_color("SELECT" <> _), do: "cyan" 

I don't think there is a need for using regular expressions. :) If we are worried about case sensitiveness, then we can define two clauses:

    defp sql_color("SELECT" <> _), do: :cyan
    defp sql_color("select" <> _), do: :cyan

But other than that we should be good to go! :D